packages/wallet/backend/src/account/service.ts

```diff
@@ -221,6 +236,28 @@ export class AccountService implements IAccountService {
         `Unable to withdraw funds from your account: ${withdrawFunds.status.message}`
       )
     }
+
+    await Transaction.query().insert({
+      accountId: account.id,
+      paymentId: withdrawFunds.data.id,
+      assetCode: account.assetCode,
+      value: await this.transformAmountBasedOnAsset(
+        args.amount,
+        account.assetId
+      ),
+      type: 'OUTGOING',
+      status: 'COMPLETED',
+      description: 'Withdraw funds'
+    })
+  }
+
+  async transformAmountBasedOnAsset(
```

Comment:
https://github.com/interledger/testnet/blob/bd82d141d1c0148b289d8a3aa896b35690331223/packages/wallet/backend/src/utils/helpers.ts#L14-L16

I think this can be used instead of having another method in the service. 🤔 Maybe `transformBalance` is not a good variable name in this use case.